packages/design-system-scripts/cli.js

```diff
@@ -163,6 +168,7 @@ yargs
         ...(argv.updateSnapshot ? ['--updateSnapshot'] : []),
         ...(argv.watch ? ['--watch'] : []),
         ...(argv.filePath ? [argv.filePath] : []),
+        ...(argv.collectCoverage ? ['--collectCoverage'] : []),
```

Comment:
Should we actually remove `design-system-scripts` from the coverage report? It's not really coverage we care deeply about, methinks, and we're wanting to replace a lot of it soon anyway.

Reply:
Had to add in the unit.config.js file, but it's done!